### ISSUE TITLE
Custom type of neurite in swc morphology

### ISSUE THREAD

**Issue by asanin-epfl** [BlueBrain/MorphIO#264]
Currently MorphIO throws an error `morphio._morphio.RawDataError` when there is a custom type but we should not decide for user what type is allowed. [SWC spec](http://www.neuronland.org/NLMorphologyConverter/MorphologyFormats/SWC/Spec.html) does not restrict custom types.

@tomdele please share your thoughts. Can we replace it with a warning at least? I can make a PR here.

```
# Use custom type: 7

 1 1  0  0 0 1. -1
 2 3  0  0 0 1.  1
 3 3  0  5 0 1.  2
 4 3 -5  5 0 0.  3
 5 3  6  5 0 0.  3
 6 7  0  0 0 1.  1
 7 7  0 -4 0 1.  6
```

**Comment by tomdele** [comment]
I think the problem here is that we need to have a corresponding enum <name, value> for each type. We cannot not tell the user which types of section he is using if we just put a number or a generic enum value.

**Comment by tomdele** [comment]
Is this a use case at the moment ?

**Comment by asanin-epfl** [comment]
It's a minor usecase for NeuroM v2. Also in general it contradicts the spec (or at least what Google gives you as the spec). Spec allows any custom neurite type. "integer value, generally continuous, starting from '1',
though this is not required." (c).

We can postpone here but it is required to fix, that's for sure.

**Comment by mgeplf** [comment]
Fixed in #274
